src/rdiff_backup/rpath.py

```diff
@@ -1677,6 +1677,8 @@ def copy_attribs(rpin, rpout):
     if Globals.change_ownership:
         rpout.chown(*map_owners.map_rpath_owner(rpin))
     if Globals.eas_write:
+        if not rpin.issym():  # make sure EAs can be written
```

Comment:
I don'T understand this change

Reply:
Is it related to an issue ?

Reply:
Ok, so it was not reported as a bug.

Thanks for the update !

Reply:
No, I changed the code to force API 201 and uncovered afew bugs while running all tests.